HMCL/src/main/resources/assets/lang/I18N.properties

```diff
@@ -308,27 +308,28 @@ folder.screenshots=Screenshots
 game=Game
 game.crash.info=Game Status
 game.crash.reason=Crash Analyzer
-game.crash.reason.duplciated_mod=Duplicated mods: %s.
-game.crash.reason.file_changed=Game file corrupted.
-game.crash.reason.gl_operation_failure=Game crashed due to some mod, shader packs, texture packs.
+game.crash.reason.duplciated_mod=The current game cannot run because of duplicate mod: %s.\n%s\nEach Mod can only be installed one, please delete the duplicate mod and try again.
+game.crash.reason.file_changed=The current game cannot run because file verification failed.\nIf you modify the Minecraft.jar, you need roll back the changes,or re-download the game.
+game.crash.reason.gl_operation_failure=Game crashed due to some mod,shader packs, texture packs.\nPlease disable the mod/shader packs/ texture packs you are using and try again.
 game.crash.reason.graphics_driver=Please update graphics driver.
-game.crash.reason.illegal_access_error=Game crashed because of %1$s.
-game.crash.reason.jdk_9=Please use Java 8.
-game.crash.reason.jvm_32bit=Please install 64-bit Java VM.
-game.crash.reason.mod=Game crashed because of mod %1$s.
-game.crash.reason.no_class_def_found_error=Game corrupted.
-game.crash.reason.no_such_method_error=Game corrupted.
-game.crash.reason.opengl_not_supported=Please update graphics driver.
-game.crash.reason.openj9=OpenJ9 JVM is not accepted.
-game.crash.reason.out_of_memory=Out of Memory.
-game.crash.reason.too_old_java=Please upgrade Java.
+The current game crashed due to a problem with your graphics driver.\n Please update your graphics driver to the latest version.\nIf your computer has a discrete graphics card,you need to check whether the game uses itegrated graphics.If so, please run run HMCL and game using discrete graphics card.If you still have these problems, you may consider getting a new graphics card or a new computer.
+game.crash.reason.illegal_access_error=Game crashed because of %1$s.\nIf you konw: %1$s,you may update or delete the mod and try again.
+game.crash.reason.jdk_9=The current game cannot run because the Java version is too high.\nYou need download and install Java 8 and set Java to 1.8 in game settings.
+game.crash.reason.jvm_32bit=If your OS is 64-bit, please install and use 64-bit Java. If your OS is 32-bit, you may reinstall 64-bit OS or change a new computer.
```

Comment:
1. [a] 64-bit
2. 'change' is not consistent with the meaning of the Chinese translation.